sockeye/inference.py

```diff
@@ -1198,7 +1223,14 @@ def _get_inference_input(self,
                 raw_constraints[j] = [data_io.tokens2ids(phrase, self.vocab_target) for phrase in
                                       trans_input.constraints]
 
-        return source, bucket_key, raw_constraints
+            if trans_input.avoid_list is not None:
+                raw_avoid_list[j] = [data_io.tokens2ids(phrase, self.vocab_target) for phrase in
+                                     trans_input.avoid_list]
+                UNK_ID = self.vocab_target[C.UNK_SYMBOL]
```

Comment:
same here. Why not 'self' it in the Translator constructor, similar to `self.start_ids` etc.